examples/point_env.py

```diff
@@ -1,30 +1,44 @@
-from rllab.envs import Env
-from rllab.spaces import Box
-from rllab.envs import Step
+from gym import Env
 import numpy as np
 
+from rllab.envs import Step
+from rllab.envs.env_spec import EnvSpec
+from rllab.spaces import Box
+
 
 class PointEnv(Env):
```

Comment:
I think we should `import gym` and then inherit from `gym.Env` so that the dependency is crystal clear.